internal/handlers/common/projection.go

```diff
@@ -28,73 +28,111 @@ import (
 // isProjectionInclusion: projection can be only inclusion or exclusion. Validate and return true if inclusion.
 // Exception for the _id field.
 func isProjectionInclusion(projection *types.Document) (inclusion bool, err error) {
```

Comment:
Let's not use named parameters as we don't really need them here.

Reply:
done